packages/cli/src/cmds/preview/depsChecker/cliAdapter.ts

```diff
@@ -3,7 +3,8 @@ import { DepsCheckerEvent, Messages } from "./common";
 import { IDepsAdapter, IDepsTelemetry } from "./checker";
 import { cliEnvCheckerLogger as logger } from "./cliLogger";
 import { cliTelemetry } from "./cliTelemetry";
-import open from "open";
+import DialogManagerInstance from "../../../userInterface";
```

Comment:
Seems not used.

Reply:
Already merged.... Will remove in the next PR.